src/main/java/pl/isa/alphateam/BoatReservationSystem.java

```diff
@@ -0,0 +1,87 @@
+package pl.isa.alphateam;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import static pl.isa.alphateam.JSONParserBoat.getListOfBoatsFromDatabase;
+import static pl.isa.alphateam.JSONParserBoat.saveBoatInDatabase;
+import static pl.isa.alphateam.JSONParserReservation.saveReservationInDatabase;
+
+public class BoatReservationSystem {
+
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+
+        // Tworzenie klienta, łodzi i rezerwacji
+        Address addressM = new Address("Polska", "Łódź", "Hetmańska", 10);
+////
+//        Boat boat = new Boat("FalconGGGG", 10, 150, true, 7778);
+        Customer customer = new Customer("MichalNNN", "Skoczylas", "1996-01-22",
+                "123456", addressM, "m.skoczylas1996@gmail.com", "babajaga");
+
+
+        // Wprowadzanie danych przez użytkownika
+        System.out.print("Select start date: (RRRR-MM-DD): ");
+        String startDateStr = scanner.nextLine();
+        LocalDate startDate = LocalDate.parse(startDateStr);
```

Comment:
Dodaj sprawdzanie czy użytkownik wprowadził poprawną datę, nigdy nie chcemy wrzucać takiego kodu na maina, w którym łatwo można wprowadzić takie dane, które przerwą działanie programu. Funkcję parse najlepiej umieścić w bloku try/catch, który złapie przypadek, kiedy użytkownik wprowadzi niepoprawną datę. Żeby trochę ułatwić, to przykładowo powinno to wyglądać mniej więcej tak:
```
LocalDate startDate = null;
while (startDate == null) {
  System.out.print("Select start date: (RRRR-MM-DD): ");
  String startDateStr = scanner.nextLine();
  try {
    startDate = LocalDate.parse(startDateStr);
  } catch (Exception e) {
    System.out.print("Wrong format");
  }
}
```